### ISSUE TITLE
CocoaPods could not find compatible versions for pod "FBSDKLoginKit" error (iOS Capactior 4)

### ISSUE THREAD

**Issue by GreenMasterTR** [capacitor-community/facebook-login#105]
Hi, I'm developing a mobile app with Capacitor 4
I get an error when using the "capacitor sync" command. (for ios)

![Screen Shot 2022-09-07 at 16 43 08](https://user-images.githubusercontent.com/17790689/188893889-877285cc-b8cf-477b-828f-7e7e85ed8b8c.png)

My project package info;
` "@capacitor/ios": "4.1.0"`
 `"@capacitor-community/facebook-login": "^4.0.0-1"`

And I applied the settings in the two required files;
`ios/App/App/AppDelegate.swift`
`ios/App/App/info.plist`

**Comment by luca-sharryland** [comment]
> Hi, I'm developing a mobile app with Capacitor 4 I get an error when using the "capacitor sync" command. (for ios)
> 
> ![Screen Shot 2022-09-07 at 16 43 08](https://user-images.githubusercontent.com/17790689/188893889-877285cc-b8cf-477b-828f-7e7e85ed8b8c.png)
> 
> My project package info; ` "@capacitor/ios": "4.1.0"` `"@capacitor-community/facebook-login": "^4.0.0-1"`
> 
> And I applied the settings in the two required files; `ios/App/App/AppDelegate.swift` `ios/App/App/info.plist`

Hi @GreenMasterTR I've found how to resolve the problem, you need to update your dependencies in Podfile. Go to /ios/App and run "pod install --repo-update".

**Comment by ciccilleju** [comment]
Hello, do you have an Arm cpu, like MacBook M1? 
If you try to build it and run on a real device, does it work? 

Because with this version i'm only able to run the simulator

**Comment by luca-sharryland** [comment]
@ciccilleju yes it works. 
Do you get errors in the console?

**Comment by ciccilleju** [comment]
@luca-sharryland everything works fine in my MacBook Pro M1, but it doesn't on my iMac Intel, but if I rollback to version 3.4.0 everything works again. 

If you read my post there are a lots of errors in console, seems like missing architecture os something like this

**Comment by luca-sharryland** [comment]
@ciccilleju 
I've installed the 4.0.0 version and after pod install --repo-update I was able to run correctly facebook plugin on an iPhone 7 real device.
I've got a MacBook Pro M1 too, unfortunately, I've not an Intel Mac for testing. 
But reading your post I think is an xcode simulator related problem, cause it breaks on a specific target in the simulator. Also for the real device, you get arm64-apple-ios11.0-simulator, I don't think it's a real device's correct target.

**Comment by ciccilleju** [comment]
> @ciccilleju 
> I've installed the 4.0.0 version and after pod install --repo-update I was able to run correctly facebook plugin on an iPhone 7 real device.
> I've got a MacBook Pro M1 too, unfortunately, I've not an Intel Mac for testing. 
> But reading your post I think is an xcode simulator related problem, cause it breaks on a specific target in the simulator. Also for the real device, you get arm64-apple-ios11.0-simulator, I don't think it's a real device's correct target.

Exactly, but i think something is missing in the dependencies,  because if I inspect the folders of version 3.4.0 and 4.0.0 they are different 

I wonder if other Intel users have the same issue

**Comment by cresjie** [comment]
@GreenMasterTR @luca-sharryland 
it might be similar to this [post](https://github.com/capacitor-community/facebook-login/issues/104), try reading the thread, we solved by upgrading the system

**Comment by jpolo** [comment]
Generally speaking the constraint with FBCoreKit to a fixed version poses a problem with peer plugins/native code that depends also on FBCoreKit.
I am using/maintaining other capacitor plugins that relies on FBCoreKit. Upgrades are really difficult in the current situation...
It would be a  great improvement if this plugin would only describe its real constraints (ex: FBCoreKit superior to version XXX, inferior to YYY) instead of pinning the FBCoreKit version.
